pkg/cmd/build/authprovider.go

```diff
@@ -84,6 +86,11 @@ func (ap *authProvider) Credentials(_ context.Context, req *auth.CredentialsRequ
 	}
 	ac, err := ap.config.GetAuthConfig(req.Host)
 	if err != nil {
+		if isErrCredentialsHelperNotAccessible(err) {
```

Comment:
In which circumstances will there be a reference to the helper but the helper is missing?